lib/modules/hosts/imgur.js

```diff
@@ -47,8 +47,8 @@ const imgur = {
 			}],
 		},
 	},
-	baseURL: 'http://imgur.com/',
-	cdnURL: '//i.imgur.com/',
+	baseURL: 'https://imgur.com/',
```

Comment:
Until imgur goes https-everywhere, this should stay http as well, or only switch to https if the user opts in.  It's only used in the links, while the mixed content errors result from the cdnURL and URLs returned from the API.